docs/index.rst

```diff
@@ -0,0 +1,10 @@
+.. include:: ../README.rst
+
+.. toctree::
+   :numbered:
+   :maxdepth: 2
+
+   usage
```

Comment:
Can you please introduce `contributing.rst` and change `usage.rst` into `gettingstarted.rst` as is the case in other REANA components?